docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,474 @@
+
+You can manage the private keys you use with AvalancheJS directly in the AvalancheJS directory by doing the following:
+
+1. Rename [`secrets.example`](https://github.com/ava-labs/avalanchejs/blob/master/examples/secrets.example) to `secrets.json`
+
+2. Add your Private Key as an object.
+
+```json
+// secrets.json
+{
+  "privateKey": "<YOUR-PRIVATE-KEY-HERE>"
+}
+```
+
+3. Import your private key into the example script.
+
+```js
+// buildExportTx-cchain-avax.ts
+import { privateKey } from "../secrets.json";
+```
+
+By default, the scripts use the AvalancheJS constant, [`DefaultLocalGenesisPrivateKey`](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts#L30) as `privKey` to sign transactions.
+
+```js
+// buildExportTx-cchain-avax.ts
+const privKey: string = `${PrivateKeyPrefix}${DefaultLocalGenesisPrivateKey}`;
+```
+
+To run AvalancheJS scripts on Mainnet or Fuji, you can simply replace this with the imported private key:
+
+```js
+// buildExportTx-cchain-avax.ts
+const privKey: string = privateKey;
+```
+
+### Step 1 - Open AvalancheJS
+
+Open your AvalancheJS directory and select the **`examples`** folder to view the source code for the examples scripts.
+
+![Image for post](/img/examples.png)
+
+### Step 2 - Modify Your Export
+
+:::info
+
+The script in this tutorial uses **X Chain <-> C Chain** transfers. Depending on which chains you want to interact with, the scripts you use could be different. Please check the table above for more information.
+
+:::
+
+<br></br>
+
+Select the **`examples/avm`** folder to view the AvalancheJS X-Chain examples. To send AVAX from the X-Chain to the C-Chain, select [`avm/buildExportTx-cchain-avax.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts)
+
+![Image for post](/img/ajs-export-cchain-res.png)
+
+**Modify Your Avalanche Network Configuration**
+
+Depending on how you started your node, you might use custom settings for your Avalanche Network
+
+_Note: See [Advanced Configurations](https://docs.avax.network/nodes/build/set-up-node-with-installer#advanced-node-configuration) to learn more about custom Node configurations_
+
+Below are the following objects in your Avalanche Network:
+
+```js
+const ip: string = "172.11.111.123";
+const port: number = 9650;
+const protocol: string = "http";
+const networkID: number = 5;
+```
+
+**IP Address**
+
+If you're using a local node:
+
+```js
+const ip: string = "localhost";
+```
+
+Or a custom IP address:
+
+```js
+const ip: string = "<YOUR-NODE-IP-ADDRESS-HERE>";
+```
+
+You can also perform this tutorial on the Avalanche Mainnet or Fuji Testnet by connecting to the [Avalanche Public API](../apis/avalanchego/public-api-server.md) server:
+
+Mainnet:
+
+```js
+const ip: string = "api.avax.network";
+```
+
+Fuji Testnet:
+
+```js
+const ip: string = "api.avax-test.network";
+```
+
+**Port Number**
+
+Default port number:
+
+```js
+const port: number = 9650;
```

Comment:
Is 9650 still the correct value here?

Reply:
unless you are running a local node. If you are using the public api server for testnet, it should be 443.